include/mysql/service_my_snprintf.h

```diff
@@ -39,40 +39,41 @@
 
   @post
   The syntax of a format string is generally the same:
-  % <flag> <width> <precision> <length modifier> <format>
+  % <flag> <width> <.precision> <length modifier> <format> <format extension>
   where everything but the format is optional.
 
   Three one-character flags are recognized:
```

Comment:
Also, we might wanna add a MariaDB line to the file’s copyright section.

Reply:
Don't worry about the copyright section. We will fix this later.

Reply:
> Don't worry about the copyright section. We will fix this later.

[GSoC Mentor LinuxJedi said during GSoC,](https://mariadb.zulipchat.com/#narrow/channel/118759-general/topic/MDEV-21978.20make.20my_vsnprintf.20to.20use.20gcc-compatible.20format/near/442716250)
> Looking at it now, but I will say that you don't need to update the copyright, particularly to that, because you don't work for the MariaDB Corporation (which also doesn't exist after the IPO).

Well… today’s ‘we’ slightly differs from the ‘we’ back then, I now work for the Corporation, ~~and K1 made the IPO a thing of the past~~.
So… what now.

Reply:
> right. please, add a second row with the MariaDB copyright and 2024

@bnestere since you’ve self-assigned testing, I (or you, be my guest)’ll bump this after your testing and before we merge.